playwright/test.spec.ts

```diff
@@ -0,0 +1,104 @@
+import { expect, test } from '@playwright/test';
+
+import {
+  loginCockpit,
+  ibFrame,
+} from './lib/lib';
+
+test.describe('test', () => {
+  test('create blueprint', async ({ page }) => {
+    await loginCockpit(page, 'admin', 'foobar');
+    // await enableComposer(page);
+    const frame = await ibFrame(page);
+
+    // image output
+    await frame.getByRole('heading', { name: 'Images About image builder' });
+    await frame.getByRole('heading', { name: 'Blueprints' });
+    await frame.getByRole('heading', { name: 'No blueprints yet' });
+    await frame.getByTestId('create-blueprint-action-emptystate').click();
+    await frame.getByRole('heading', { name: 'Image output' });
+    await frame.getByTestId('checkbox-guest-image').click();
+    await frame.getByRole('button', { name: 'Next', exact: true }).click();
+
+    // fs
+    await frame.getByRole('heading', { name: 'Additional packages' });
```

Comment:
Should this be FS heading?

Reply:
oh wait, why does that work then O.o  if it's looking for packages twice, maybe the nav in the sidebar is playing tricks.